indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -286,10 +340,17 @@
             self.$categoryResults.html('');
             _.each(categories, function(category) {
                 var $result = self._buildSubcategory(category, true);
-                $result.find('.icon-wrapper').append($('<i>', {
-                    class: 'icon-search',
-                    title: $T.gettext("Search result")
-                }));
+                if (category.is_favorite) {
+                    $result.find('.icon-wrapper').append($('<i>', {
+                        class: 'icon-star',
```

Comment:
Technically, it IS properly aligned 😛 
![image](https://cloud.githubusercontent.com/assets/1579899/16488761/5bc55f22-3ed2-11e6-866c-5cfbdcc0a2b9.png)